### ISSUE TITLE
Hope this helps everyone!

### ISSUE THREAD

**Issue by truff1ep1g** [checkra1n/BugTracker#861]
I found this tucked away in a thread.  After days of trying everything to get past Linux installs, and wrong sized usb drives and seeing these guys on YouTube getting more and more hits because they are posting tutorials to claim to have fixed all theses errors. And regarding the same videos over and over. This little jewel was hidden away and no one I saw anywhere else Was suggesting this as a fix. Even in the other threads. On here it was virtually unknown or Unsuggested.  https://github.com/foxlet/bootra1n/blob/master/README.md

Please if you are having errors 77, 21 or whatever. Getting stuck at download mode or anywhere else try this. It’s simple. Even for someone who is sorta new.   It’s easy and you might learn something you didn’t know in the process. But I want the guys responsible to get the credit.

**Comment by billabongbruno** [comment]
lol
This is EXACTLY what I referred to earlier, except I went into much more detail.

Got my post deleted for "piracy".

Funny how this one stays up.

Oh well, it's all for the best.

Thank you anyway.

**Comment by truff1ep1g** [comment]
I posted it everywhere because I felt like many of us were in the same error camp. I tried every fix I could find. This worked first try on every device I have

**Comment by nergzd723** [comment]
@billabongbruno, actually you reffered into using ra1nusb and old version of checkra1n. And AFAIK bootra1n works only on Intel since it's booting Darwin kernel and macOS in general.

@truff1ep1g, unfortunately bootra1n doesn't work for any of my PC's. Same issue. It's strange that Coffee Lake chipset does not work either.
edit: wrote bootra1n instead of ra1nusb

**Comment by truff1ep1g** [comment]
I don’t have intel my brother. Amd    A PoS hp amd laptop at that. With 4 gb ram and like 1.8 cpu.  What you using intel I take it?

**Comment by billabongbruno** [comment]
@nergzd723 , no, actually it was neither one of those. I talked about something else and even though my guide did mention the Intel file, the AMD is also available and it's the exact same procedure.

@truff1ep1g , AMD works fine too.
